styles/confirmEmail.module.css

```
@@ -0,0 +1,6 @@
+.center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
```

We have a specific font style for all of our text. You can add this to a div that surrounds all the elements or separately to each text div.

```suggestion
font-family: var(--mainFont) !important;
```